auth/shared-signal-receiver/handlers/request-handler.ts

```diff
@@ -6,21 +8,84 @@ import { handleAccountPurgedRequest } from './account-purged-handler';
 import type { APIGatewayProxyResult } from 'aws-lambda';
 import { generateResponse } from '../response';
 import { ReasonPhrases, StatusCodes } from 'http-status-codes';
+import { CognitoError } from '../errors';
+import { logMessages } from '../log-messages';
+import { verifyUsername } from '../cognito/verify-users';
 
 interface Handler {
   schema: z.ZodType;
   /* eslint-disable @typescript-eslint/no-explicit-any */
   handle: (data: any) => Promise<APIGatewayProxyResult>;
+  schemaName: string;
+  allowedChangeType?: string;
 }
 
 const handlers: Handler[] = [
   {
     schema: credentialChangeSchema,
     handle: handleCredentialChangeRequest,
+    schemaName:
+      'https://schemas.openid.net/secevent/caep/event-type/credential-change',
+    allowedChangeType: 'update',
```

Comment:
I'm not sure what this property is for, in `credential-change-handler.ts` we're checking if it's a valid change type already.